website/docs/patterns/multi-select-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,63 @@
+!!!
+
+## Selected count
+
+Display the **selected count** to communicate how many rows or results have been selected in the _entire_ data set. This includes rows that are outside of the current view in the Table (e.g., in a different page if using [Pagination](/components/pagination)), and should be represented using a numerical value.
+
+![Example of the selected count](/assets/patterns/multi-select-patterns/selected-count.png)
+
+Pair this with a **message** to clearly communicate that a selection has been made. This can be a simple generic message stating "{number in current selection} selected", or can be more specific to the type of object in the data set (e.g., a user, cluster, workspace, etc).
+
+![Generic and specific messaging](/assets/patterns/multi-select-patterns/types-of-messages.png)
+
+## Total count
+
+If technically feasible, communicate the selected count as a portion of the **total count**, or total number of results in the Table. This can give the user a better overall snapshot of what they have selected and how the selection compares to the entire data set. This can also more effectively communicate that there may be results that are selected across multiple pages.
+
+![Total count of items and count of selected items](/assets/patterns/multi-select-patterns/total-count-selected-count.png)
+
+!!! Info
+
+Depending on how the data within a Table is fetched or if [cursor-based pagination](/components/pagination#numbered-vs-compact) is used, it may not be possible to get the total number of results in a data set.
+!!!
+
+## Bulk actions
+
+The act of selecting multiple results in a Table implies that the the user wants to edit, remove, or transform these results in some way. Consolidate these actions in a [List](/components/dropdown#list) toggled by a `small` [Dropdown](/components/dropdown) inline with the selected count, message, and total count.
+
+![Example of bulk actions](/assets/patterns/multi-select-patterns/bulk-actions.png)
+
+When working with instances of a Table where only a single action is necessary (e.g., bulk deleting a selection), use a `small` [Button](/components/button) in place of the Dropdown. We recommend using only `secondary` or `critical` Buttons in this scenario depending on the intent of the action.
+
+![Example of a single action](/assets/patterns/multi-select-patterns/single-action.png)
+
+### Editing results
+
+With a selection made, consider using a [Modal](/components/modal) to perform bulk edits or updates to parameters of the selected results. This is generally appropriate for editing five parameters or less, for more complex scenarios consider shifting the editing experience to a separate page.
```

Comment:
> consider shifting the editing experience to a separate page

I'm not sure about this. When they leave this page and come back to it after editing, what happens? Do they come back to this page? Is the multi-select state for each row retained? Does a success toast appear? What happens if the editing action fails?

Reply:
I can add some additional clarifying content here, but this editing experience in it's own page was validated by TJ on some Vault work, that is how he's expecting to handle it because it's more complex then something that would live in a Modal.

I would imagine: 

1. You're taken to another page to edit. This page acts as sort of confirm/cancel  experience
2. Once editing is complete, you're redirected back to the table. Seeing a Toast would add a good confirmation reinforcement. If the editing action fails, I would either expect a critical Toast upon redirection, _or_ for some sort of validation to happen on the editing page to avoid that.
3. I would expect the selection to be persisted after editing is complete.

I can add all of these points in as an example scenario, do you think this is a reasonable pattern?